LogIn/Auth/AuthManager.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  AuthManager.swift
+//  LogIn
+//
+//  Created by Aleksei Permiakov on 11.01.2022.
+//
+
+import FirebaseAuth
+import Combine
+
+final class AuthManager {
+    
+    static let shared = AuthManager()
+    private init() {}
+    
+    var isLoggedIn = CurrentValueSubject<Bool, Never>(false)
+    
+    var userEmail: String? {
+        Auth.auth().currentUser?.email
+    }
+    
```

Comment:
Take a look https://www.donnywals.com/using-promises-and-futures-in-combine/ to see how to wrap completion API's with combine